__tests__/integration.test.js

```diff
@@ -120,6 +137,14 @@ describe("GET /api/articles", () => {
         expect(response.status).toBe(404);
       });
   });
+    test('GET 404: sends an appropriate status and error message when non-existent topic', () => {
```

Comment:
might be a good test idea to see what should happen with a topic that exists with no articles (e.g. 'paper')